stacker/blueprints/base.py

```diff
@@ -179,6 +200,11 @@ def resolve_variable(var_name, var_def, provided_variable, blueprint_name):
     var_type = var_def.get("type")
     value = validate_variable_type(var_name, var_type, value)
 
+    allowed_values = var_def.get('allowed_values')
+    valid = validate_allowed_values(allowed_values, value)
+    if not valid:
+        raise UnallowedValue(var_name, value, allowed_values)
```

Comment:
Yea, I was just using the `UnallowedValue` error to construct the message which probably isn't worth it